explorer/src/providers/accounts/history.tsx

```diff
@@ -166,7 +166,9 @@ export function useFetchAccountHistory() {
           limit: 25,
         });
       } else {
-        fetchAccountHistory(dispatch, pubkey, cluster, url, { limit: 25 });
+        fetchAccountHistory(dispatch, pubkey, cluster, url, { 
+          limit: before?.data?.fetched.length || 25 
```

Comment:
This isn't the fix I was looking for. We don't need to refetch the transactions that were already fetched

Reply:
Ok. Anyway, I was happy to help)